site/layouts/partials/newsletter-form.html

```diff
@@ -5,8 +5,6 @@
     <p class="hidden">
       <label>Don’t fill this out if you're human: <input name="bot-field"></label>
     </p>
-    <legend class="f5 f4-ns mb1 black-80">Subscribe for updates:</legend>
```

Comment:
These checkboxes look a little odd without a description of what they are for, no?

Reply:
Wanted to get rid of the checkmarks entirely, my bad.